src/dev/precommit_hook/casing_check_config.js

```diff
@@ -165,7 +165,7 @@ export const TEMPORARILY_IGNORED_PATHS = [
   'src/core/server/core_app/assets/favicons/safari-pinned-tab.svg',
   'test/functional/apps/management/exports/_import_objects-conflicts.json',
   'x-pack/legacy/plugins/index_management/public/lib/editSettings.js',
-  'x-pack/legacy/plugins/license_management/public/store/reducers/licenseManagement.js',
+  'x-pack/legacy/platform/plugins/shared/license_management/public/store/reducers/licenseManagement.js',
```

Comment:
I think that we can remove these 2 lines, although we can do it on a separate PR.